test/integration/test_pages_redirects.js

```diff
@@ -20,35 +20,8 @@ test('redirects "/" to guides.18f.gov', (t) => {
 
 const pageConfigs = lib.getPageConfigs(fs.readFileSync(PAGES_FILE, 'utf-8'));
 
-const subdomainConfigs = pageConfigs.filter(pc => !pc.toDomain);
```

Comment:
Every pageConfig has a `.toDomain`, with the default being `18f.gov`, so this filter actually results in `subdomainConfigs` being an empty array. That means the `.forEach` down on line 26 doesn't actually run, so the tests within it were never run.

There's no need to separate those with a "custom" `toDomain` and those with the default since the behavior is the same for either case.